weblate/utils/stats.py

```diff
@@ -833,6 +833,31 @@ def calculate_item(self, item):
         self.store(item, result)
 
 
+class UserStats(BaseStats):
```

Comment:
I'm still not sure that introducing the stats object for user is way to go. It mostly just mirrors the fields in the database and introduces risk of inconsistencies.

This is very different from the translation stats where heavy aggregation is going on and it saves a lot of resource to calculate that.

Reply:
Then should I just add it directly in user model as a dictionary?

Reply:
My biggest concern is whether we need UserStats at all. I currently doesn't do any expensive calculations, so I'd say it's not needed for now.
You can use directly the Profile attributes in the UserStatisticsSerializer without need for intermediate object.